PendingReleaseNotes

```diff
@@ -707,3 +707,11 @@ errorcode32_t is converting, internally, the error code from host to ceph, when
 when decoding, resulting having LINUX codes on the wire, and HOST code on the receiver.
 All CEPHFS_E* defines have been removed across Ceph (including the python binding).
 Relevant tracker: https://tracker.ceph.com/issues/64611
+
+* RGW: Persistent bucket Notifications will use queues with multiple shards instead of one queue. Number of shards
```

Comment:
Notifications lowercase please.  

suggest:

The number of shards is specified by the central config option `rgw_bucket_persistent_notif_num_shards`.